clients/admin-ui/src/features/common/nav/v2/NextBreadcrumb.tsx

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable tailwindcss/no-custom-classname */
+/* eslint-disable no-param-reassign */
```

Comment:
I kind of agree with this eslint rule. I think it's better to not modify parameters, as mutation can have side-effects and I also find it less confusing when you have an original vs a modified version of the object.

Something like this maybe: `const modifiedItem = { ...item };`. What do you think?